lang/java/reef-runtime-standalone/src/main/java/org/apache/reef/runtime/standalone/driver/Container.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.reef.runtime.standalone.driver;
+
+import com.jcraft.jsch.Session;
+import org.apache.reef.annotations.audience.Private;
+
+import java.io.File;
+import java.util.List;
+
+/**
+ * Represents a Container: A slice of a machine.
+ * <p>
+ * In the case of the local resourcemanager, this slice is always the one of the machine where the job was submitted.
```

Comment:
This comment is a bit confusing as it is talking about `the local resourcemanager`. Also I find that its only implementation is `ProcessContainer`, which makes me wonder whether we can remove this class altogether.